### PR TITLE
Update timezone defaults to ET

### LINKED ISSUE #124: Update time zone to ET and make it clear in the UI

### PR DESCRIPTION
**Pull request by stvnrlly**
This sets a default timezone of 'America/New_York', but allows it to be configured via `TIMEZONE` in the environment.

Right now, this just goes by ET, so it's prominent for the user. A next step here would be to convert in the browser based on the user's timezone. We might consider updating the time selection interface at the same time.

This closes #124

### CODE REVIEW

File: app/main/views/notifications.py [hunk 1]
```diff
@@ -59,40 +46,20 @@ def view_notification(service_id, notification_id):
...
-            # This is because currently we do not pull the letter from S3 when showing preview.
-            # Instead, we generate letter preview based on the letter template and personalisation.
-            # Additionally, when a templated letter is sent via the api and the personalisation pushes the
-            # page count over 10 pages, it takes a while for validation status to come through.
-            # Checking page count here will enable us to show the error message even if the letter is not
-            # fully processed yet.
-            error_message = get_letter_validation_error(
-                "letter-too-long", [1], page_count
-            )
+    page_count = get_page_count_for_letter(notification['template'], values=personalisation)
```
Comment (rahearn) [inline app/main/views/notifications.py:49]: [question] why are we leaving this letter code in? It seemed like you're just deleting other letter stuff

File: app/main/views/performance.py [hunk 2]
```diff
@@ -12,8 +12,8 @@
 @main.route("/performance")
 def performance():
     stats = performance_dashboard_api_client.get_performance_dashboard_stats(
-        start_date=(datetime.utcnow() - timedelta(days=7)).date(),
-        end_date=datetime.utcnow().date(),
+        start_date=(datetime.now(current_app.config['PY_TIMEZONE']) - timedelta(days=7)).date(),
```
Comment (rahearn) [inline app/main/views/performance.py:15]: [question] can this logic be used above where there's a `TODO` to get a time delta in local timezone?

File: app/utils/letters.py [hunk 3]
```diff
@@ -1,47 +1,6 @@
-from datetime import datetime, timedelta
-
-import pytz
-from dateutil import parser
 from flask import url_for
```
Comment (rahearn) [inline app/utils/letters.py:1]: [question] can we not delete the whole module?

File: tests/app/main/forms/test_choose_time_form.py [hunk 4]
```diff
@@ -4,26 +4,28 @@
...
-    assert choices[38] == ('2016-01-03T01:00:00', 'Sunday at 1am')
+    assert choices[38] == ('2016-01-03T06:00:00', 'Sunday at 1am ET')
 
     # Monday
-    assert choices[84] == ('2016-01-04T23:00:00', 'Monday at 11pm')
-    assert choices[85] == ('2016-01-05T00:00:00', 'Monday at midnight')
+    assert choices[62] == ('2016-01-04T06:00:00', 'Monday at 1am ET')
+    assert choices[80] == ('2016-01-05T00:00:00', 'Monday at 7pm ET')
+    # assert choices[84] == ('2016-01-05T04:00:00', 'Monday at 11pm')
+    # assert choices[85] == ('2016-01-05T05:00:00', 'Monday at midnight')
```
Comment (rahearn) [inline tests/app/main/forms/test_choose_time_form.py:28]: [question] Why the comment out here?

Comment (stvnrlly) [reply tests/app/main/forms/test_choose_time_form.py:28]: Hmmm, let's see if I can get that working correctly.

File: app/main/views/notifications.py (hunk 1 above)
Comment (rahearn) [reply app/main/views/notifications.py:49]: Nah, I'm good with a couple of smaller PRs

File: app/utils/letters.py (hunk 3 above)
Comment (rahearn) [reply app/utils/letters.py:1]: Nevermind on this, address that as a separate letters PR

File: app/main/views/performance.py (hunk 2 above)
Comment (stvnrlly) [reply app/main/views/performance.py:15]: Yes! Thanks for finding my `TODO`

File: tests/app/main/forms/test_choose_time_form.py (hunk 4 above)
Comment (stvnrlly) [reply tests/app/main/forms/test_choose_time_form.py]: Okay, now future sending options are calculated correctly based on local time. Now I need to make sure that they _also_ get sent to the database correctly

Comment (stvnrlly) [reply tests/app/main/forms/test_choose_time_form.py]: I think we're all set now; take another look and let me know what you think.

**Review by rahearn — APPROVED** [review]

**Pull request merged**